src/main/java/com/google/sps/servlets/CasesDataServlet.java

```diff
@@ -64,11 +74,17 @@ public void init() {
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setCharacterEncoding(ENCODING);
     response.setContentType(CTYPE);
-    if (reportsJson != null) {
-      response.getWriter().println(reportsJson);
-    }
+    response.getWriter().println(reportsJson);
```

Comment:
What I meant offline is that you can write something like:
response.getWriter().println(getReportsJson);

Then in the test you can use:
when(servlet.getReportJson()).thenReturn(xxx);  // xxx is a temporary/sample data you create in the test.